sssom/writers.py

```diff
@@ -554,3 +604,18 @@ def _get_separator(serialisation: Optional[str] = None) -> str:
             f"Unknown table format: {serialisation}, should be one of tsv or csv"
         )
     return sep
+
+
+def _resolve_url(prefixed_url_str: str, prefix_map: PrefixMap) -> str:
+    if not prefixed_url_str:
+        return prefixed_url_str
+
+    prefix_url = prefixed_url_str.split(":")
+    if len(prefix_url) != 2:
+        return prefixed_url_str
+    else:
+        return _resolve_prefix(prefix_url[0], prefix_map) + prefix_url[1]
```

Comment:
Just for my understanding, can I please get an example of what a typical `prefixed_url_str` would look like?

Reply:
@syphax-bouazzouni I don not see a response here.